symfony/framework-bundle/4.2/config/bootstrap.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
This should be removed in all files as we are not using strict_types in Symfony.

Reply:
@fabpot removed.

I would be happy to know the reasons. There is an issue (https://github.com/symfony/symfony/issues/28423) discussing that it can't be used for parts of the framework which are backported to older versions. But it suggests that it may be used for new components, which won't be backported to <4.0 versions.